pixels-turbo/pixels-worker-common/src/main/java/io/pixelsdb/pixels/worker/common/WorkerCommon.java

```diff
@@ -53,15 +52,15 @@
  */
 public class WorkerCommon
 {
-    private static final Logger logger = LoggerFactory.getLogger(WorkerCommon.class);
+    public static final int rowBatchSize;
+    private static final Logger logger = LogManager.getLogger(WorkerCommon.class);
     private static final PixelsFooterCache footerCache = new PixelsFooterCache();
     private static final ConfigFactory configFactory = ConfigFactory.Instance();
+    private static final int pixelStride;
```

Comment:
Don't change this file except for the log manager.

Reply:
Okay, I will restore